src/families/algorand/specs.js

```diff
@@ -23,7 +23,7 @@ const minBalanceNewAccount = parseCurrencyUnit(currency.units[0], "0.1");
 // Spendable balance for a non-ASA account
 const getSpendableBalance = (maxSpendable) => {
   maxSpendable = maxSpendable.minus(minFees);
-  invariant(maxSpendable.gt(minFees), "spendable balance is too low");
+  invariant(maxSpendable.gt(minFees), "Spendable balance is too low");
```

Comment:
Naïve question: shouldn't this test be performed _before_ the modification of `maxSpendable`? Or the test be `gt(0)`?

Reply:
I agree with you. I will fix it.

Reply:
Done.